app/webpack/stylesheets/components/_phase_banner.scss

```diff
@@ -0,0 +1,33 @@
+$laa-uat-colour: govuk-colour("orange");
+$laa-staging-colour: govuk-colour("turquoise");
+$laa-dev-colour: govuk-colour("pink");
```

Comment:
Do you need these variables? Couldn't you apply govuk colours directly?

Reply:
@naseberry It would be possible to apply the govuk colours directly but I thought it made sense to use the variables as they are referenced in a couple of places (it is also how it was implemented on FALA https://github.com/ministryofjustice/fala/blob/f126531e74a9787d161b3c28813063a431b27c2b/fala/assets-src/sass/main.scss)

Reply:
@kmahern multiple reference is valid. FALA's implementation works but it is about making ours better. I would suggest the follow:

- Rename  variables from `$laa-` to `$app-`, why? because LAA has no defined colour palette
- If these variables will be used elsewhere, suggest moving them to a separate file, i.e `../stylesheets/settings/_colours.scss`
- MInor issue, use single quotes instead of double quotes.

PS - Hope you don't mind if we copy this feature for CCCD

Reply:
👍  have renamed variables $app- as suggested and amended to use single quotes. There are no plans to use the variables other than for border-bottom-color and background-color on the govuk-phase-banner so I have left them in the _phase_banner.scss file for now.